src/main/java/io/vertx/core/parsetools/RecordParser.java

```diff
@@ -68,7 +69,20 @@
    * @param output  handler that will receive the output
    */
   static RecordParser newDelimited(String delim, Handler<Buffer> output) {
-    return RecordParserImpl.newDelimited(delim, output);
+    return RecordParserImpl.newDelimited(delim, null, output);
+  }
+
+  /**
+   * Like {@link #newDelimited(String)} but wraps the {@code stream}, the stream handlers will be set/unset
```

Comment:
Minor, I would change:
`but wraps the {@code stream}, the stream handlers`
to
`but wraps the {@code stream}. The {@code stream} handlers`
for readability